packages/core/src/reporters/html/single-file-template/single-file-template.ts

```diff
@@ -0,0 +1,25 @@
+import { readFileSync } from 'fs';
+
+import { mutationTestReportSchema } from '@stryker-mutator/api/report';
+
+export function singleFileTemplate(report: mutationTestReportSchema.MutationTestResult) {
+  return `<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <script>
+    ${readFileSync(require.resolve('mutation-testing-elements/dist/mutation-test-elements.js')).toString('utf-8')}
+  </script>
+</head>
+<body>
+  <img class="stryker-image" alt="Stryker" src="stryker-80x80.png" style="position: fixed; right: 0; top: 0; z-index: 10">
+  <mutation-test-report-app titlePostfix="Stryker">
+    Your browser doesn't support <a href="https://caniuse.com/#search=custom%20elements">custom elements</a>.
+    Please use a latest version of an evergreen browser (Firefox, Chrome, Safari, Opera, etc).
+  </mutation-test-report-app>
+  <script>
+    document.querySelector('mutation-test-report-app').report = ${JSON.stringify(report)};
```

Comment:
This actually won't work if you have a `file` in the result which contains a `</script>` because HTML parsing takes precedence over JS parsing. 

I'm pretty sure Stryker.NET solved this by simply splitting the string there and concatenating it. So something like

```js
JSON.stringify(report).replace(/<\/script>/g, '</sc" + "ript>');
```

This is maybe not the best way, and might not work in all circumstances, so I'm very open to suggestions here.

Reply:
we can go with this simple solution for now. I mean, users will have two options: Bulletproof multifile and basic one file

Reply:
I found the workaround:

https://github.com/stryker-mutator/stryker-net/blob/e847896228aad54809ad0db0c188ef0a5d7d7d9f/src/Stryker.Core/Stryker.Core/Reporters/JsonReporter/JsonReport.cs#L67

Probably do it the exact same way 🤷‍♂️